source/target/freescale/rapid_iot/flash_blob_k64.c

```diff
@@ -0,0 +1,90 @@
+/**
+ * @file    flash_blob_k64.c
+ * @brief   Flash algorithm for the k64f
+ *
+ * DAPLink Interface Firmware
+ * Copyright (c) 2009-2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "flash_blob.h"
+
+static const uint32_t K64F_FLM[] = {
```

Comment:
Why is an additional K64F flash algo added?

Reply:
Hexiwear has K64F and KW40Z.
Rapid-IoT has K64F and KW41Z.

Depending on the switch settings, user can flash to either K64F or KW4x, Hence we read the SDID register from the target to load the appropriate flash algo.